packages/wonder-blocks-timing/src/util/types.js

```diff
@@ -161,12 +172,7 @@ export interface IScheduleActions {
      * @returns {ITimeout} A interface for manipulating the created timeout.
      * @memberof IScheduleActions
      */
-    timeout(
-        action: () => mixed,
-        period: number,
-        autoSchedule?: boolean,
-        resolveOnClear?: boolean,
-    ): ITimeout;
+    timeout(action: () => mixed, period: number, options?: Options): ITimeout;
```

Comment:
Having strongly typed options with named values should make it a lot easier to interpret calls instead of looking into what the last two boolean values mean.